tests/Fixtures/e2e/Save_PHPUnit_Bootstrap_File/infection.json

```diff
@@ -0,0 +1,12 @@
+{
+    "timeout": 25,
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "summary": "infection-log.txt"
+    },
+    "bootstrap": "./custom-autoload.php"
```

Comment:
should we add this case to documentation? (When project has custom autoloaded, it should be added to `bootstrap` section of infection config)

Reply:
Isn't the goal of this PR to make sure that we use the custom bootstrap file in phpunit, rather than hard coding it. 

So if we add it here to the custom infection bootstrap file that does not prove that it is correctly included/used.

Reply:
Sorry, I didn't get what you mean

> Isn't the goal of this PR to make sure that we use the custom bootstrap file in phpunit, rather than hard coding it.

Yes, this is a goal of this PR. Previously we always included `vendor/autoload` in PHPUnit process, now we include that file, that was in the original `phpunit.xml`

> So if we add it here to the custom infection bootstrap file that does not prove that it is correctly included/used.

Here, in `infection.json` we add custom autoload to make Infection work itself (not PHPUnit process that infection runs). Without it, our ReflectionVisitor won't be able to autoload classes that are not autoloaded by composer's autoloader. You can run added e2e test to see what I mean and remove this line

Reply:
Ah i understand whats going on now.

In that case, yes, we should add it to the documentation.

Reply:
added documentation https://github.com/infection/site/pull/36